src_assets/common/assets/web/config.html

```diff
@@ -482,37 +439,54 @@ <h2 class="accordion-header">
             5.1/7.1 surround sound and muting host audio.
           </div>
         </div>
-        <!--Adapter Name -->
+
+        <!-- Adapter Name -->
         <div class="mb-3" v-if="platform === 'windows'">
           <label for="adapter_name" class="form-label">Adapter Name</label>
           <input type="text" class="form-control" id="adapter_name" placeholder="Radeon RX 580 Series"
             v-model="config.adapter_name" />
-          <div class="form-text" v-if="platform === 'windows'">
-            Manually specify a GPU to use for capture. If unset, the GPU is chosen automatically.<br />
-            <b>We strongly recommend leaving this field blank to use automatic GPU selection!</b><br />
-            Note: This GPU must have a display connected and powered on.<br />
-            The appropriate values can be found using the following command:<br />
+          <div class="form-text">
+            Manually specify a GPU to use for capture. If unset, the GPU is chosen automatically.<br>
+            <b>We strongly recommend leaving this field blank to use automatic GPU selection!</b><br>
+            Note: This GPU must have a display connected and powered on.<br>
+            The appropriate values can be found using the following command:<br>
             <pre>tools\dxgi-info.exe</pre>
           </div>
         </div>
-        <!--Output Name -->
+        <div class="mb-3" v-if="platform === 'linux'">
+          <label for="adapter_name" class="form-label">Adapter Name</label>
+          <input class="form-control" id="adapter_name" placeholder="/dev/dri/renderD128" v-model="config.adapter_name" />
+          <div class="form-text">
+            Manually specify a GPU to use for capture.<br>
+            <pre>ls /dev/dri/renderD*  # to find all devices capable of VAAPI</pre>
+            <pre>
+              vainfo --display drm --device /dev/dri/renderD129 | \
+                grep -E "((VAProfileH264High|VAProfileHEVCMain|VAProfileHEVCMain10).*VAEntrypointEncSlice)|Driver version"
+            </pre>
+            Replace ``renderD129`` with the device from above to lists the name and capabilities of the device.<br>
+            To be supported by Sunshine, it needs to have at the very minimum:
+            <i>VAProfileH264High   : VAEntrypointEncSlice</i>
+          </div>
+        </div>
```

Comment:
This was on the va-api tab previously, and it was the only option there. I thought it made more sense to put on the A/V tab... and not sure it applies to only va-api anyway?